src/HealthChecks.AzureSearch/DependencyInjection/AzureSearchHealthCheckBuilderExtensions.cs

```diff
@@ -0,0 +1,28 @@
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace HealthChecks.AzureSearch.DependencyInjection;
+
+public static class AzureSearchHealthCheckBuilderExtensions
+{
+    private const string NAME = "azuresearch";
+
+    public static IHealthChecksBuilder AddAzureSearch(

```

Comment:
Please add xml comments here and for other public APIs.

Reply:
Sure, will do that!